de.metas.adempiere.adempiere/base/src/main/java/de/metas/adempiere/service/impl/AddressBuilder.java

```diff
@@ -86,11 +88,10 @@ public AddressBuilder()
 	 * @param userBlock
 	 * @return
 	 */
-	public String buildAddressString(final I_C_Location location, boolean isLocalAddress, String bPartnerBlock, String userBlock)
+	public String buildAddressString(final I_AD_Org org, final I_C_Location location, final String language, boolean isLocalAddress, String bPartnerBlock, String userBlock)
```

Comment:
More, on a second look i am wondering why not having the "org" and "language" as fields,
so introducing fluent setAD_Org(org) and setLanguage(adLanguage) methods would make the code much more readable instead of having so many arguments here...